src/spz/util/init_db.py

```diff
@@ -10,7 +10,9 @@
 from jsonschema import validate, ValidationError, SchemaError
 
 from spz import app, db
-from spz.models import *  # Keep this import, otherwise the create_all call won't any models at all
+from spz.models import Degree, Graduation, Origin, Language, Course, User
+# Make sure that create_all works for all models (even ones that might be added in the future)
+from spz.models import *  # noqa
```

Comment:
Isn't this a duplicate import?

Reply:
Sort of...
Without the explicit imports, flake8 will go mad ("Class may be undefined, or defined from star imports").
Without the wildcard import, we need to add an import for every single database model. Otherwise the models table might be missed in `db.create_all()`.

Reply:
Ah okay, makes sense.